plugins/logistics.cpp

```diff
@@ -391,7 +439,7 @@ static const struct BadFlags {
     uint32_t get_bad_flags() {
         df::item_flags flags;
         #define F(x) flags.bits.x = true;
-        F(forbid); F(garbage_collect); F(hostile); F(on_fire);
+        F(garbage_collect); F(hostile); F(on_fire);
```

Comment:
this will cause forbidden items to start being processed by the other processors. I think you need to modify the other `can_designate` functions to compensate

Reply:
Here's what I have in mind, do you concur?
- forbidden items should **should be marked** for other operations
- forbidden items **should be unmarked** for other operations
- logistics stats should ignore forbidden items

Been slower than expected to get back to this but I still  hope to have this and the rest of the feedback fixed up tomorrow (US-Sunday).

Reply:
I think the general expectation is that forbidden items will be untouched by all tools (other than the ones that specifically deal with forbidding/unforbidding).

stats can maybe call out forbidden items separately

Reply:
I realized I was overthinking this and imagining that toggling off melting would undesignate items formerly designated for melting. So as you suggested I just incorporated the forbid into `can_designate`